include/rocksdb/advanced_options.h

```diff
@@ -126,19 +126,28 @@ struct CompressionOptions {
   // Default: 0.
   uint32_t zstd_max_train_bytes;
 
+  // When the compression options are set by the user, it will be set to "true".
```

Comment:
let's make it explicit that the user must set `enabled=true` in order for `bottommost_compression_opts` to be used. And even if `compression_opts.enabled=false`, we will still use it.